applications/nrf_desktop/bootloader_dfu.rst

```diff
@@ -185,6 +189,20 @@ For an example of a bootloader Kconfig configuration file defined by the applica
   Both mentioned firmware upgrade methods are not used simultaneously by any of the configurations.
   For example, the ``nrf52840dk/nrf52840`` board in ``mcuboot_smp`` file suffix uses only the background DFU and does not enable the serial recovery feature.
 
+MCUboot bootloader on nRF54L
+----------------------------
+
+nRF54L SoC Series allows achieving improved security and shorter boot times by using hardware cryptography in the MCUboot immutable bootloader.
+The |NCS| allows using hardware cryptography for ED25519 signature (``SB_CONFIG_BOOT_SIGNATURE_TYPE_ED25519``) on nRF54L SoC Series.
+
+The following sysbuild Kconfig options allow to improve security further:
+
+* ``SB_CONFIG_BOOT_SIGNATURE_TYPE_PURE`` - This option enables using pure signature of the image (verifying signature directly on image, rather than hash of the image).
+  The option cannot be used if external memory is used for the secondary image slot.
+* ``SB_CONFIG_MCUBOOT_SIGNATURE_USING_KMU`` - This option enables using Key Management Unit (KMU) to store keys for signature verification instead of compiling in key data from a file.
```

Comment:
```suggestion
* ``SB_CONFIG_MCUBOOT_SIGNATURE_USING_KMU`` - This option enables using Key Management Unit (KMU) to store keys for signature verification instead of compiling key data from a file.
```

Reply:
This one comes from https://github.com/nrfconnect/sdk-mcuboot/blob/main/boot/zephyr/Kconfig#L318.

Maybe we could use `instead of compiling key data into MCUboot bootloader image`?

Reply:
Yeah, that sounds better